ext/standard/streamsfuncs.c

```diff
@@ -1643,11 +1643,14 @@ PHP_FUNCTION(sapi_windows_vt100_support)
 		php_stream_cast(stream, PHP_STREAM_AS_FD, (void*)&fileno, 0);
 	}
 	else {
-		zend_type_error(
-			"%s() was not able to analyze the specified stream",
-			get_active_function_name()
-		);
-		RETURN_THROWS();
+		if (argc > 1) {
```

Comment:
Could you please fix the `UNKNOWN` default value?

Reply:
You mean by changing the type to `?bool`? Sure! But in my opinion it would have been better to have two functions instead.

Reply:
Yes, I meant the ?bool type. And of course I agree with you that these functions having get-and-set functionality in the same time are horrible.

Reply:
Changed to `?bool`. :)